ansible_bender/builders/buildah_builder.py

```diff
@@ -150,7 +150,10 @@ def configure_buildah_container(container_name, working_dir=None, env_vars=None,
         for k, v in annotations.items():
             config_args += ["--annotation", "%s=%s" % (k, v)]
     if user:
-        config_args += ["--user", user]
+        if os.getuid() != 0:
+            print("Setting user in rootless mode will create issues")
+        if os.getuid() == 0:
```

Comment:
```suggestion
        else:
```
no need to do the syscall again